device/device.go

```diff
@@ -507,8 +525,9 @@ func (device *Device) BindUpdate() error {
 	device.net.stopping.Add(len(recvFns))
 	device.queue.decryption.wg.Add(len(recvFns)) // each RoutineReceiveIncoming goroutine writes to device.queue.decryption
 	device.queue.handshake.wg.Add(len(recvFns))  // each RoutineReceiveIncoming goroutine writes to device.queue.handshake
+	batchSz := netc.bind.BatchSize()
```

Comment:
batchSize

Reply:
Updated in https://github.com/WireGuard/wireguard-go/pull/64/commits/37142f9b30ec7fafcf80ebd7a158824203ce053f.